test/test.pl

```diff
@@ -216,7 +304,16 @@ sub test_cmd
         }
         else
         {
-            failed($opts,$test,"The outputs differ:\n\t\t$$opts{path}/$args{out}\n\t\t$$opts{path}/$args{out}.new");
+            if ( exists($args{exp}) && !-e "$$opts{path}/$args{out}" )
```

Comment:
The second part of this will always be false due to the condition for the surrounding `if` statement.